plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -208,7 +214,7 @@ private static void createPomXml(IProject project, AppEngineProjectConfig config
     String sdkVersion = getCurrentVersion(
         "com.google.appengine", //$NON-NLS-1$
         "appengine-api-1.0-sdk", //$NON-NLS-1$
-        "1.9.62"); //$NON-NLS-1$
+        "1.9.63"); //$NON-NLS-1$
```

Comment:
I think it's worth pulling these artifact constants (including the BOM artifact too) out into explicit constants rather than have them scattered in the code.

Reply:
Makes sense. Can you file an issue for that?

Reply:
#2958